CHANGELOG.md

```diff
@@ -41,14 +40,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Support for returning output from policy changes invocations
 - Fatal exception is raised when this module is run on Python2
 
-## [0.0.1] - 2019-05-01
-
-The first tagged version.
+## 0.0.1 - 2019-05-01
+### Added
+- The first tagged version.
 
 [Unreleased]: https://github.com/conjurinc/conjur-api-python3/compare/v0.1.0...HEAD
-[0.1.0]: https://github.com/conjurinc/conjur-api-python3/compare/v0.0.5...0.1.0
-[0.0.5]: https://github.com/conjurinc/conjur-api-python3/compare/v0.0.4...0.0.5
-[0.0.4]: https://github.com/conjurinc/conjur-api-python3/compare/v0.0.3...0.0.4
-[0.0.3]: https://github.com/conjurinc/conjur-api-python3/compare/v0.0.2...0.0.3
-[0.0.2]: https://github.com/conjurinc/conjur-api-python3/compare/v0.0.1...0.0.2
-[0.0.1]: https://github.com/cyberark/conjur-api-python3/tree/v0.0.1
+[0.1.0]: https://github.com/conjurinc/conjur-api-python3/compare/v0.0.5...v0.1.0
+[0.0.5]: https://github.com/conjurinc/conjur-api-python3/compare/v0.0.4...v0.0.5
+[0.0.4]: https://github.com/conjurinc/conjur-api-python3/compare/v0.0.3...v0.0.4
+[0.0.3]: https://github.com/conjurinc/conjur-api-python3/compare/v0.0.2...v0.0.3
+[0.0.2]: https://github.com/conjurinc/conjur-api-python3/compare/v0.0.1...v0.0.2
```

Comment:
Let's add the first release link:
```
[0.0.1]: https://github.com/cyberark/conjur-api-python3/release/tag/v0.0.1
```

Reply:
I believe that the link for v0.0.1 should use `releases` instead of `release`:
```
https://github.com/cyberark/conjur-api-python3/releases/tag/v0.0.1
```

Reply:
Fixed